src/assertions/text.spec.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env mocha */
+/* eslint-disable padded-blocks, no-unused-expressions, max-len, no-console */
+
+import './text';
+import { shallow, mount } from 'enzyme';
+import React from 'react';
+
+let wrapper;
+const TextFixture = () => (
+  <div className="special burger">Content here. More content</div>
+);
+
+describe('Should enzyme add check text', () => {
+  const shouldEnzyme = should;
+  it('should have new method text', () => {
+    shouldEnzyme.should.have.property('containsText');
+  });
+  [shallow, mount].forEach(method => {
```

Comment:
Created a new function to handle this functionality called `eachRenderMethod`. [See example](https://github.com/rkotze/should-enzyme/blob/master/src/assertions/class-name.spec.js#L16)